core/cmd.go

```diff
@@ -142,22 +143,37 @@ func (ch *CommandHandler) GetUsers() ([]*model.User, error) {
 	return users, err
 }
 
-func (ch *CommandHandler) GetCustomThemeList() ([]string, error) {
-	files, err := ioutil.ReadDir(`themes`)
+func (ch *CommandHandler) GetThemes() ([]model.Theme, error) {
+	// get internal themes
+	internalThemes := server.GetInternalThemes()
+
+	// get custom themes
+	files, err := os.ReadDir(`themes`)
 	if err != nil {
-		log.Println(err)
-		return nil, errorsx.NewFormattedError(http.StatusNotFound, errors.New("failed to read themes directory"))
+		return internalThemes, nil
```

Comment:
In which cases would we fail to read? Im thinking about a case where there are custom themes but we just failed to read, do we want to notify users in such a case?

Reply:
When they haven't created the `themes` directory we should just return the internal themes